examples/gin/README.md

```diff
@@ -2,10 +2,28 @@
 
 This is an example of basic Gin app with Airbrake middleware that reports route stats.
 
-## How to run API
+## How to run Example API
+
+Copy the example in your testing(eg. ginbrake) folder.
+
+Insert your project ID and project key in main.go file
+
+run:
 
 ```bash
-go run *.go -env=production -airbrake_project_id=123456 -airbrake_project_key=FIXME
+mkdir ginbrake
+cd ginbrake
+go mod init ginbrake
+go mod tidy
+go run main.go
 ```
 
-Visit http://127.0.0.1:8080/hello/{name}
+Valid endpoints for testing are:
+
+`curl "http://127.0.0.1:3000/date" -H 'api-key: d4b371692d361869183d92d84caa5edb8835cf7d'`
```

Comment:
```suggestion
`curl "http://localhost:3000/date" -H 'api-key: d4b371692d361869183d92d84caa5edb8835cf7d'`
```

Reply:
Also, put all the curl statements in a single code block rather than separate code blocks:

```
curl "..."
curl "..."
curl "..."
```

Instead of:

`curl ...`

`curl ...`

`curl ...`